ajax/libs/leaflet-groupedlayercontrol/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "leaflet-groupedlayercontrol",
+  "filename" : "leaflet.groupedlayercontrol.min.js",
+  "version": "0.6.0",
+  "description": "A Leaflet layer control with support for grouping overlays together",
+  "main": "dist/leaflet.groupedlayercontrol.min.js",
+  "directories": {
+    "example": "example"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "//git:github.com/ismyrnow/leaflet-groupedlayercontrol",
```

Comment:
Invalid `target`. You can take a look at [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).